### DISCUSSION TITLE
Recommended budget GPU for LLM experimentation

### DISCUSSION THREAD

**Discussion by msfasha** [oobabooga/text-generation-webui#5782] [Q&A]
I want to get a budget GPU for LLM training/fine tuning/experimentation.
I am considering:
- a single rtx 4070 ti super 16gb
- or a single rtx 4080 16gb
- or some dual lower level gpus setting, e.g. dual rtx 4070 12gb ram

Any thoughts, advices or other recommendations?

**Comment by jepjoo** [comment]
Don't know much about training and all that, but a used RTX 3090 generally offers good bang for buck.

Should be better than those 40 series GPU's that you listed since it has more memory (and higher memory bandwidth).

**Comment by TiagoTiago** [comment]
I wonder how much a laptop 3080 (16GB, more than desktop 3080's for some reason) is going for nowadays.....
